BaragonUI/app/components/serviceDetail/util.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import fuzzy from 'fuzzy';
+
+// [T], PosInt -> [[T]]
+// [1, 2, 3, 4, 5, 6, 7], 2 -> [[1, 2], [3, 4], [5, 6], [7]]
+const chunk = (arr, size) => {
```

Comment:
rather than having a separate util file in the serviceDetail folder, we should put these in the more global utils file. Seems plenty of these may be reused in components for other pages